src/Umbraco.Core/HealthChecks/Checks/LiveEnvironment/RuntimeModeCheck.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Umbraco.
+// See LICENSE for more details.
+
+using Microsoft.Extensions.Options;
+using Umbraco.Cms.Core.Configuration.Models;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Extensions;
+
+namespace Umbraco.Cms.Core.HealthChecks.Checks.LiveEnvironment;
+
+/// <summary>
+///     Health check for the recommended production configuration for the runtime mode.
+/// </summary>
+[HealthCheck(
+    "8E31E5C9-7A1D-4ACB-A3A8-6495F3EDB932",
+    "Runtime Mode",
+    Description = "Health check for the recommended production configuration for runtime mode.",
```

Comment:
It looks like the description is supposed to be a user-facing explanation of  the Health Check, rather than a description of the check itself 🤷.

How about this instead?
```suggestion
    Description = "The Production Runtime Mode disables development features and checks that settings are configured optimally for production.
```